WalletWasabi/WabiSabi/WabiSabiCoordinator.cs

```diff
@@ -160,7 +160,7 @@ public void BanDoubleSpenders(object? sender, Transaction tx)
 
 	private bool IsWasabiCoinJoinLookingTx(Transaction tx) =>
 		tx.RBF == false
-		&& tx.Inputs.Count >= Config.MinInputCountByRound
```

Comment:
Should maybe some check be added somewhere in code that `MinInputCountByBlameRound` is not bigger than `MinInputCountByRound`? It can be if human error is made in `~/.walletwasabi/backend/WabiSabiConfig.json`.

Reply:
That change doesn't solve the problem but hide it instead, and that's always wrong. The thing is that no software can do anything with wrong inputs, that's why we have to validate everything that comes from the user (external sources in general) as soon as possible, preferably in the border of the system. So, if the user makes a mistake and sets the multiplier to 80 instead of 0.8 the software has to fail immediately with an error "Multiplier out of range blh blah blah".

Reply:
I think backend should brutally fail at start if `MinInputCountByBlameRoundMultiplier` is greater (or not smaller) than `MinInputCountByRoundMultiplier`. But that's out of the scope of this PR.